mir-ci/mir_ci/fixtures/servers.py

```diff
@@ -17,7 +18,8 @@ class ServerCap(Flag):
     FLOATING_WINDOWS = auto()
     DRAG_AND_DROP = auto()
     DISPLAY_CONFIG = auto()
-    ALL = FLOATING_WINDOWS | DRAG_AND_DROP | DISPLAY_CONFIG
+    SCREENCOPY_BANDWIDTH = auto()
+    ALL = FLOATING_WINDOWS | DRAG_AND_DROP | DISPLAY_CONFIG | SCREENCOPY_BANDWIDTH
```

Comment:
```suggestion
    SCREENCOPY = auto()
    ALL = FLOATING_WINDOWS | DRAG_AND_DROP | DISPLAY_CONFIG | SCREENCOPY
```

These should be capabilities, not test suites.